tests/random/background-fetch-content.html

```diff
@@ -260,6 +262,15 @@ <h1 id="result-name"></h1>
               options.downloadTotal = downloadTotal;
             }
 
+            if (icons.checked) {
+              options.icons = [];
+              options.icons.push({
+                "src": "resources/icons/17.png",
```

Comment:
One more nit: The `src` needs to start with a /.

Right now it'll try to load `tests.peter.sh/random/resources/icons/17.png`, which doesn't work. It needs to be an absolute path, thus `tests.peter.sh/resources/icons/17.png`.